packages/asset-uploader/jest.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
please reuse something for both sdk and this one, I think it makes sense to have 2 different once, e.g. one for react, one for node, the node one doesn't need majority of stuff

Reply:
I thought this is pure node code, so all the jsx jazz etc is not needed

Reply:
oh wait you don't have jsx, its just ts and we need to run ts through esbuild, I think your jest config is perfectly fine if that works, lets just move it into a separate package so that we can reuse it for other node packages